recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set name = "xeus_octave" %}
+{% set name = "xeus-octave" %}
```

Comment:
Naming the package xeus-octave instead of xeus_octave to be more consistent with the other xeus kernels